polystores/clients/aws_client.py

```diff
@@ -42,6 +42,16 @@ def get_aws_verify_ssl(keys=None):
     return False if env == 'False' else env
```

Comment:
This needs to be changed as well, since now you will get `False`, you can just return `env` directly.

Reply:
Fixed!